docs/research/user-experience/xmr-btc-user-interface-validation.md

```diff
@@ -0,0 +1,117 @@
+|   |  |  | use a Bitcoin transaction to give funds to the swap application | TM |
+|   |  |  | use a Monero transaction to give funds to the swap application | TM |
+|   |  |  | explicitly sign each transaction | TM |
+|  Discovery | How do users discover trading partners? | Do users care about privacy? | go to website and take price from there | T |
+|   |  |  | set up website (publicly) to advertise price (and connection information) | M |
+|   |  |  | open "random" (tor) website found on various media (forums, chat) to access a single market maker. | T |
+|   |  |  | configure Tor for trading | TM |
+|   |  | Do users care about P2P? | use a centralized service to find makers | TM |
+|   |  |  | user a decentralized service to find makers | TM |
+|  Software Setup |  |  | download software (swap execution daemon) before being able to do a swap | T |
+|   |  |  | keep a GUI/CLI open for the whole length of the swap execution | T |
+|   |  |  | keep a computer running (that hosts the daemon) for the whole length of the swap execution | T |
+|   |  |  | keep the browser open for the whole length of a swap | T |
+|  Protocol |  |  | have different steps (locking first vs second) depending on the direction of the swap | TM |
+
+## Prototypes
+
+In the initial project description we distinguished product `A` a single market-maker product and product `B` a product including peer-to-peer discovery and multiple makers.
+
+```
+P ... Prototype that showcases a complete swap setup flow.
+D ... Prototype that focuses on a specific detail of swap setup / execution.
+
+{}-A ... Prototype for product A (single market maker)
+{}-B ... Prototype for product B (decentralized trading platform)
+```
+
+Example:
+
+`D-A2-1`: Mock showing detail 1 for prototype `P-A1`
+
+### Figma Links
+
+* [P-A1](https://www.figma.com/proto/QdvmbRYuBpEpFI3D1R4qyM/XMR-BTC_SingleMaker_LowFidelity?node-id=54%3A4894&viewport=1503%2C-52%2C0.5576764941215515&scaling=min-zoom): Webpage for discovery, CLI for execution
+* [P-A2](https://www.figma.com/proto/QdvmbRYuBpEpFI3D1R4qyM/XMR-BTC_SingleMaker_LowFidelity?node-id=7%3A4377&viewport=696%2C-250%2C0.362735778093338&scaling=min-zoom): Webpage for discovery, GUI for execution
+* [D-A2-1](https://www.figma.com/proto/QdvmbRYuBpEpFI3D1R4qyM/XMR-BTC_SingleMaker_LowFidelity?node-id=235%3A1374&viewport=1336%2C-1825%2C0.7878535389900208&scaling=min-zoom): GUI swap execution steps for `send` `BTC`, `receive` `XMR`
+* [D-A2-2](https://www.figma.com/proto/QdvmbRYuBpEpFI3D1R4qyM/XMR-BTC_SingleMaker_LowFidelity?node-id=128%3A8016&viewport=1404%2C-1158%2C0.66261225938797&scaling=min-zoom): GUI swap execution steps for `send` `XMR`, `receive` `BTC`
+
+### Mapping of Prototype to validation criteria
+
+|  **User is happy to...** | **Actor** | **P-A1** | **P-A2** | **D-A2-1** | **D-A2-2** |
```

Comment:
I would suggest to merge both tables to make our life easier and not have to ensure they are in sync.
I am happy to have a crack at it if you already tried and do not think there is an elegant way to achieve that.

Reply:
It actually is one table in the google sheet where I manage this - I added you.
My plan was to manage the data in the gsheet, and once it reaches a state where it adds value to publish it update both tables in this document.
I use a gsheet plugin to generate the markdown tables, only takes a few seconds to generate the code and then it's copy paste. I know it's not optimal to have the data in a gsheet and here, but it's just way easier to fill and manage this in a gsheet.

Reply:
I kept the two table approach for now. If we produce a high number of prototypes it will be hard to keep adding columns to one complete markdown table. If you feel strong about this please ping me.

Reply:
I am not going to tell you how to do it :) It was merely a suggestion to make your live easier ;)